### PR TITLE
US-3.7: Split documents portlet to recent documents, favorite documents and shared with me documents portlets

### PR DESCRIPTION
**Pull request by AzmiTouil**

### CODE REVIEW

File: digital-workplace-webapps/webpack.common.js [hunk 1]
```diff
@@ -11,9 +11,9 @@ let config = {
     profileStats: './src/main/webapp/vue-app/profile-stats/main.js',
     tasks: './src/main/webapp/vue-app/tasks/main.js',
     events: './src/main/webapp/vue-app/events/main.js',
-    documents: './src/main/webapp/vue-app/documents/main.js',
-    wallet: './src/main/webapp/vue-app/wallet/main.js',
-    perkstore: './src/main/webapp/vue-app/perkstore/main.js'
+    sharedDocuments: './src/main/webapp/vue-app/documents-shared/main.js',
```
Comment (azayati) [inline digital-workplace-webapps/webpack.common.js]: Rename the folders to sharedDocuments recentDocuments ...

File: digital-workplace-webapps/webpack.dev.js [hunk 2]
```diff
@@ -6,7 +6,7 @@ const webpackCommonConfig = require('./webpack.common.js');
 const app = 'digital-workplace';
 
 // add the server path to your server location path
-const exoServerPath = "/exo-server";
+const exoServerPath = "/home/exo/Digitale-Workplace/platform-6.0.x-SNAPSHOT";
```
Comment (azayati) [inline digital-workplace-webapps/webpack.dev.js]: Why we need this modification ?

File: digital-workplace-webapps/src/main/webapp/documents-favorites.html [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<div class="VuetifyApp">
```
Comment (azayati) [inline digital-workplace-webapps/src/main/webapp/documents-favorites.html]: Rename the html files to sharedDocuments.html recentDocuments.html ...

File: digital-workplace-webapps/src/main/webapp/WEB-INF/conf/digital-workplace/portal/dw/pages.xml [hunk 4]
```diff
@@ -117,20 +117,49 @@
...
+                  <show-info-bar>false</show-info-bar>
+                </portlet-application>
+              </container>
+              <container template="system:/groovy/portal/webui/container/UIContainer.gtmpl">
+                <portlet-application>
+                  <portlet>
+                    <application-ref>digital-workplace</application-ref>
+                    <portlet-ref>digitalWorkplaceSharedDocuments</portlet-ref>
+                  </portlet>
+                  <title>Shared with me</title>
```
Comment (azayati) [inline digital-workplace-webapps/src/main/webapp/WEB-INF/conf/digital-workplace/portal/dw/pages.xml]: Is it possible to get the container title internationalized ?

File: digital-workplace-webapps/webpack.common.js (hunk 1 above)
Comment (AzmiTouil) [reply digital-workplace-webapps/webpack.common.js]: Done

File: digital-workplace-webapps/webpack.dev.js (hunk 2 above)
Comment (AzmiTouil) [reply digital-workplace-webapps/webpack.dev.js]: reverted

File: digital-workplace-webapps/src/main/webapp/documents-favorites.html (hunk 3 above)
Comment (AzmiTouil) [reply digital-workplace-webapps/src/main/webapp/documents-favorites.html]: Done

**Pull request merged**